### ISSUE TITLE
RC Spot Check: Molecules and Light 1.4.0-rc.3

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/QA#90]
After #84 and #89, Molecules and Light 1.4.0-rc.3 is ready for a spot check. @mattpen and @jonathanolson helped look into https://github.com/phetsims/perennial/issues/79, and the correct images are now being generated.

[Link to sim](https://www.colorado.edu/physics/phet/dev/html/molecules-and-light/1.4.0-rc.3/molecules-and-light_en.html)

[Link to iFrame](https://www.colorado.edu/physics/phet/dev/html/molecules-and-light/1.4.0-rc.3/molecules-and-light_en-iframe.html)

**Issues to Verify**
Please test the following issues and check them off after addressing. If they are resolved, close them. If not, they should be updated.
- [x] https://github.com/phetsims/molecules-and-light/issues/175

Changes for the above are targeted for Edge, so please verify that all is well on that platform. SVG is now the root renderer for molecules-and-light on Edge, so please thoroughly test the sim on Edge and be on the lookout for strange rendering artifacts. But for sanity please also spot check on a few platforms.

If any new issues are found, please note them in https://github.com/phetsims/molecules-and-light/issues and reference this issue.

#89 had top priority, re-adding label.

**Comment by phet-steele** [comment]
Good to go.

**Comment by jessegreenberg** [comment]
Thanks @phet-steele and team. Sim has been redeployed.
